mac

```diff
@@ -176,6 +178,8 @@ brew_tap 'caskroom/cask'
 brew_install_or_upgrade 'brew-cask'
 brew_tap 'caskroom/versions'
 
+brew_cask_install 'alfred'
+brew cask install 'dash'
```

Comment:
I don't think either of these tools are very widely-used at Vermonster. Alfred is a bit of a personal-preference thing, and Dash is a paid program whose trial version gradually renders itself unusable (through constant wait times) if you don't pay for it. I've been using http://devdocs.io as a replacement with some success.